.github/CODEOWNERS

```
@@ -1,5 +1,62 @@
-/.github/ @elastic/doc-leads
-/.github/workflows/ @elastic/docs-engineering
+# Unless a later match takes precedence, someone on the docs team must review all PRs to this repo
+*                                           @elastic/docs
 
-/serverless/ @elastic/platform-docs
-/serverless/serverless-changelog.asciidoc @elastic/docs
+# Templates, workflows, and config
+/.github/                                   @elastic/doc-leads
+/.github/workflows/                         @elastic/docs-engineering
+/acrolinx/                                  @elastic/doc-leads
+
+# The docs
+/cloud-account/                             @elastic/admin-docs
+/deploy-manage/                             @elastic/admin-docs
+/explore-analyze/                           @elastic/core-docs
```

```suggestion
/explore-analyze/                           @elastic/core-docs @elastic/experience-docs 
```
This is a mix of dev docs and experience docs